link-verifier/action.yml

```diff
@@ -94,13 +97,17 @@ runs:
         args+=" --include-file-types ${file_types}"
       fi
 
+      # Many of the FreeRTOS-Repos include a link to MISRA's website. This website
+      # now has a CAPTCHA landing page, as such always exclude it from this check.
+      touch allowList.txt
+      echo "https://www.misra.org.uk/misra-c" >> allowList.txt
+      echo "https://www.misra.org.uk" >> allowList.txt
```

Comment:
Idea here is to make this fix more "portable" as any repos that are already using the above parameters would need an individual PR

Adding this here means that we only need to make this change here.